.forms.yml

```diff
@@ -0,0 +1,243 @@
+        key: aws_default_region
+        widget: auto_complete
+        type: string
+        values: ["eu-north-1","ap-south-1","eu-west-3","eu-west-2","eu-west-1","ap-northeast-2","ap-northeast-1","sa-east-1","ca-central-1","ap-southeast-1","ap-southeast-2","eu-central-1","us-east-1","us-east-2","us-west-1","us-west-2"]
+        default: eu-west-1
+      - name: "Terraform storage bucket"
+        description: "AWS S3 bucket name to store terraform remote state file."
+        key: terraform_storage_bucket_name
+        widget: simple_text
+        type: string
+        default: ($ organization_canonical $)-terraform-remote-state
+
+    Repos:
+      - name: "Config repository"
+        key: config_git_repository
+        type: string
+        widget: cy_crs
+        description: "Git repository URL containing the config of the stack."
+        default: "git@github.com:MyUser/config-ec2.git"
+        required: true
+      - name: "Config git key"
+        description: "SSH key pair to fetch the config Git repository."
+        key: config_git_private_key
+        type: string
+        widget: cy_cred
+        default: "((ssh_config.ssh_key))"
+        required: true
+      - name: "Config branch"
+        description: "Branch of the config Git repository."
+        key: config_git_branch
+        widget: cy_branch
+        type: string
+        description: "The branch used by the config repository"
+        source: config_git_repository
+        default: "master"
+        required: true
+
+  terraform:
+    vm:
+      - name: "AMI enable most recent"
+        description: "If more than one result is returned, use the most recent AMI"
+        key: module.vm.ami_most_recent
+        widget: switch
+        type: boolean
+        default: true
+      - name: "AMI name"
+        description: "The name of the AMI."
+        key: module.vm.ami_name
+        widget: simple_text
+        type: string
+        default: "debian-stretch-*"
+        required: true
+      - name: "AMI virtualization type"
+        description: "The AMI virtualization type."
+        key: module.vm.ami_virtualisation_type
+        widget: radios
+        type: string
+        default: "hvm"
+        values:
+          - "hvm"
+          - "paravirtual"
+      - name: "AMI image architecture"
+        description: "The AMI image architecture"
+        key: module.vm.ami_architecture
+        widget: radios
+        type: string
+        default: "x86_64"
+        values:
+          - "x86_64"
+          - "i386"
+          - "arm64"
+      - name: "AMI type of root device volume"
+        description: "The AMI type of the root device volume."
+        key: module.vm.ami_root_device_type
+        widget: radios
+        type: string
+        default: "ebs"
+        values:
+          - "ebs"
+          - "instance-store"
+      - name: "AMI owners"
+        description: "The AMI owners, used to limit search."
+        key: module.vm.ami_owners
+        widget: dropdown
```

Comment:
Maybe switch it to an auto_complete widget (to let also user enter their own owners)